usaspending_api/conftest.py

```diff
@@ -43,6 +58,60 @@
 baker.generators.add("usaspending_api.common.custom_django_fields.BooleanFieldWithDefault", lambda: False)
 
 
+def pytest_collection_modifyitems(session: pytest.Session, config: pytest.Config, items: List[pytest.Item]) -> None:
```

Comment:
You should document the parameters here with a description.

Reply:
Done